[Ready for review] Add contributor personas & pathways (#470)
alan-turing-institute/the-turing-way: personas.md

# Personas & Pathways

Personas are character sketches of fictional contributors and users who might do something with the Turing Way.
Following the [Mozilla Open Leadership training](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/bring-on-contributors-using-personas-and-pathways/), we created personas for contributors to the Turing Way based on participants in past trainings and events, and have created personas for users based on our target audiences.

> This will help us in our development of the Turing Way community such as through the Readme and Contributing Guidelines.

Reviewer: ```suggestion
This will help us in our development of the Turing Way community such as through the [README](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md) and [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md).
```

We have also imagined how our personas might interact with our project – the "pathway" they take to get involved and to sustain involvement which comprises a few key phases:

1. Discovery - How they first hear about the project or group.
2. First Contact - How they first engage with the project or group, their initial interaction.
3. Participation - How they first participate or contribute.
4. Sustained Participation - How their contribution or involvement can continue.
5. Networked Participation - How they may network within the community.
6. Leadership - How they may take on some additional responsibility on the project, or begin to lead.

> Below you’ll find a range of personas and pathways, showing how their interactions with the Turing Way can improve and enrich the project.

Reviewer: ```suggestion
Below you'll find a range of personas and pathways, showing how their interactions with the Turing Way can improve and enrich the project.
```

Each example is meant to suggest ways you might interact with, learn from, and contribute to the Turing Way so hopefully one resonates with you and empowers you to get involved!

> If you do not relate to any of the personas described and are struggling to figure out how to get involved or use this resource, please open an issue or get in contact on our Gitter channel letting us know what the barriers are!

Reviewer: ```suggestion
If you do not relate to any of the personas described and are struggling to figure out how to get involved or use this resource, please open an issue or get in contact on our [Gitter channel](https://gitter.im/alan-turing-institute/the-turing-way) letting us know what the barriers are!
```

They are passionate about open data and using data for good, but also understand it can be more complex than that.
They attend data science meetups in the city, but are interested in finding a community working towards improving how research is conducted.
They have experience with GitHub and contributing to open projects.

1. **Discovery** - Alex first hears about the Turing Way via Twitter ([#TuringWay](https://twitter.com/search?q=%23TuringWay&src=typed_query)).
2. **First Contact** - They land on the project's README and look for the contributing guidelines to see where their skills can be applied.
3. **Participation** - They find an issue asking for help on writing the Credit for Reproducible Research chapter and add a few paragraphs on how to properly cite research software.
4. **Sustained Participation** - After constructive feedback and collaboration with the Turing Way team, Alex follows the community on social media and applies to attend the Manchester Book Dash.
5. **Networked Participation** - They invite other friends and colleagues from the Rstats community to get involved, who have expertise on some of the book topics.

> 6. **Leadership** - Alex volunteers to write the chapter on Scoping a data project - RSEs that they found requested in the [Book Skeleton](https://github.com/alan-turing-institute/the-turing-way/blob/master/book_skeleton.md).

Reviewer: ```suggestion
6. **Leadership** - Alex volunteers to write the chapter on "Scoping a data project for RSEs" that they found requested in the [Book Skeleton](https://github.com/alan-turing-institute/the-turing-way/blob/master/book_skeleton.md).
```

5. **Networked Participation** - Noor tweets a link to the Turing Way during a conversation about reproducible research and retweets a response from another expert which leads to a longer conversation. 
6. **Leadership** - After submitting their PhD, and unsure what they want to do next, Noor volunteers to co-ordinate further research into the chapter by reaching out to experts in the space.

## User personas

### Early career researcher

> Still to write.

1. **Discovery** -
2. **First Contact** -
3. **Participation** -
4. **Sustained Participation** -
5. **Networked Participation** -
6. **Leadership** -

### Supervisor / PI

> Still to write.

1. **Discovery** -
2. **First Contact** -
3. **Participation** -
4. **Sustained Participation** -
5. **Networked Participation** -
6. **Leadership** -

### Funder / publisher / admin

> Still to write.

1. **Discovery** -
2. **First Contact** -
3. **Participation** -
4. **Sustained Participation** -
5. **Networked Participation** -
6. **Leadership** -

---

> We have created these personas and pathways for contributors and users to guide the development of the Turing Way following [Mozilla Open Leadership training](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/bring-on-contributors-using-personas-and-pathways/). This personas.md file has remixed and reused material from the [Open Leadership Framework GitHub repository](https://github.com/mozilla/open-leadership-framework/blob/master/personas.md) (License: [CC-BY-SA](https://creativecommons.org/licenses/by/4.0/)).

Reviewer: ```suggestion
We have created these personas and pathways for contributors and users to guide the development of the Turing Way following [Mozilla Open Leadership training](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/bring-on-contributors-using-personas-and-pathways/).
This personas.md file has remixed and reused material from the [Open Leadership Framework GitHub repository](https://github.com/mozilla/open-leadership-framework/blob/master/personas.md) (License: [CC-BY-SA](https://creativecommons.org/licenses/by/4.0/)).
```
